client/src/components/Login.jsx

```diff
@@ -1,9 +1,9 @@
-import React from 'react'
+import React from "react";
```

Comment:
please delete `Login` folder, since we now have `Amplify`'s UI for login. Thanks!